docs/team/radiankrisno.md

```diff
@@ -34,18 +31,18 @@ Given below are some of my notable contributions to the project.
     * What it does: Allows the user to clear only records of patients, doctors, or activities in GoMedic.
     * Justification: This feature is really helpful as sometimes you only want to discard information about patients, doctors, or activities.
       This feature is also very helpful during manual testing.
-    * Highlights: Clear activities handle all of the expected causations, e.g. when you clear all patients, all appointments related to the patients will be removed as well.
+    * Highlights: `clear` command handles all the expected causation, e.g. when you clear all patients, all appointments related to the patients will be removed as well.
```

Comment:
Btw, isnt this Seng Leng part for the e.g, since he made the appointment after the clear commands, can check with @ngsengleng